src/app/main/main.controller.js

```diff
@@ -2,70 +2,5 @@
 
 angular.module('uvpBuilderWeb')
   .controller('MainCtrl', function ($scope) {
```

Comment:
this does not follow https://github.com/johnpapa/angular-styleguide#style-y022 it has to be

```javascript
function HomeController() {}

angular
  .module('uvpBuilderWeb')
  .controller('HomeController', HomeController)
```